en/docs/setup/migrating-what-has-changed.md

```diff
@@ -1,387 +1,277 @@
+
+```toml
+[[allowed_user_stores]]
+class = "org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager"
+```
+
+## JWT validation at introspection
+
+JWT validation at introspection is enabled by default in WSO2 IS 5.11.0. The server identifies a JWT token at the introspection endpoint by attempting a JWT token parsing. If identified, introspection is performed by treating the token as a JWT access token.
+
+**Note** that enabling this feature will validate the token using the available JWT token validator for instance, in the default pack, it will use the `Default JWT token validator`. So any valid and parsable JWT token should be able to be successfully validated against the available JWT token validator in the system. 
+
+However, if the server issues custom JWT tokens which fail validation with the available JWT token validator, the existing flow can break. In a migrated setup, this condition can fail in scenarios such as:
+
+- There are custom token issuers that issue valid and parsable JWT tokens which fail the default JWT token validator shipped with the pack. e.g., a custom token not having the `jti` claim.
+
+- Both a custom JWT token validator and the default JWT token validator co-exist in the system. There also exists an invalid custom token which is successfully validated with the default validator but fails to be validated with the custom validator. The token may still pass the introspection validation.
+
+Therefore, it is important to verify that the JWT token validator can avoid the scenarios mentioned above. If not, it needs to be handled by deploying a custom JWT token validator that will avoid the issues mentioned above.
+
+To revert to the previous behavior and disable this feature, add the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.
 
-These properties can be managed by adding the below configuration to `deployment.toml` file
 ```toml
 [oauth]
-prompt_login_consent = false
-prompt_logout_consent = false
+enable_jwt_token_validation_during_introspection=false
 ```
 
-## WSDL of the Application Management Service has been changed
-In IS 5.10.0, there are significant changes has been introduced to the WSDL of the Application Management Admin Service 
-with the addition of new properties to the service provider model object. Therefore SOAP API consumers should 
-regenerate the stub or upgrade the dependency version of the stub.
+## Disabled appending params to OAuth2/OIDC error responses
+
+Appending additional parameters to the OAuth2/OIDC error response has been disabled by default. As a result, the parameters 'sp' and 'tenant_domain' which was set to the callback url in error responses in previous WSO2 IS versions will also not be available. If any configured error param needs to be sent back to the callback url in an error response, add the following config to the `deployment.toml` file.
+
+```toml
+[oauth.allow_additional_params_from_error_url]
+enable = true
+```
```

Comment:
@sherenewso2 this config should be changed as:

```
[oauth]
allow_additional_params_from_error_url = false
```

Reply:
fixed with https://github.com/wso2/docs-is/pull/2168